notebooks/en/mlflow_ray_serve.ipynb

```diff
@@ -0,0 +1,1079 @@
+{
```

Comment:
<p>Nice "motivation" section to really clarify the intent of this notebook</p><p> </p><p>"...can meaningfully drive down..."</p><p> </p><p>"...may not have the benefit of existing infrastructure to form..."</p><p> </p><p>"...deploy a new model, the better."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done!